containers/src/Data/IntMap/Internal.hs

```diff
@@ -2329,7 +2329,7 @@ lookupMax :: IntMap a -> Maybe (Key, a)
 lookupMax Nil         = Nothing
 lookupMax (Tip k v)   = Just (k,v)
 lookupMax (Bin p l r) = Just $! lookupMaxSure (if signBranch p then l else r)
-{-# INLINE lookupMax #-}
+{-# INLINE lookupMax #-} -- See Note [Inline lookupMin] in Data.Set.Internal
```

Comment:
Looks like a space snuck in?

Reply:
Maybe something's wrong with my eyes but I don't see it... could you point it out?

Reply:
I think you replaced the extra spaces with comments, or I made a mistake. Anyway, it looks good.